pkgs/by-name/io/iowow/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "iowow";
+  version = "1.4.16";
+
+  src = fetchFromGitHub {
+    owner = "Softmotions";
+    repo = "iowow";
+    rev = "v${version}";
+    hash = "sha256-4TL/ZNVtRc4+xk1qj0pnUMIA2kyMsBcBqpMeEvvbGUg=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  postPatch = ''
+    substituteInPlace src/tmpl/libiowow.pc.in \
+      --replace '$'{prefix}/@CMAKE_INSTALL_LIBDIR@ @CMAKE_INSTALL_FULL_LIBDIR@ \
+      --replace '$'{prefix}/@CMAKE_INSTALL_INCLUDEDIR@ @CMAKE_INSTALL_FULL_INCLUDEDIR@ \
+      --replace '$'{prefix}/@CMAKE_INSTALL_BINDIR@ @CMAKE_INSTALL_FULL_BINDIR@
+  '';
```

Comment:
Please submit upstream

Reply:
OK, I will submit it

Reply:
With a link to https://github.com/jtojnar/cmake-snips/#concatenating-paths-when-building-pkg-config-files